deploy/deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+## For debugging
+# redirect stdout/stderr to a file
+exec &> log.out
+
+
+echo 'Simple Web Server for testing the deployment'
+sudo apt update -y
+sudo apt install apache2 -y
+sudo systemctl start apache2
+echo """
+<h1 style='color:#f09764; text-align:center'>
+    OpenMined First Server Deployed via Terraform
+</h1>
+""" | sudo tee /var/www/html/index.html
+
+echo 'Setup Miniconda environment'
+
+sudo wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+sudo bash miniconda.sh -b -p miniconda
+sudo rm miniconda.sh
+export PATH=/miniconda/bin:$PATH > ~/.bashrc
+conda init bash
+source ~/.bashrc
+conda create -y -n pygrid python=3.7
+conda activate pygrid
+
+echo 'Install poetry...'
+pip install poetry
+# sudo curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+# source $HOME/.poetry/env
+
+echo 'Install GCC'
+sudo apt-get install python3-dev -y
+sudo apt-get install libevent-dev -y
+sudo apt-get install gcc -y
+
+echo 'Clonning PyGrid'
+
+git clone https://github.com/OpenMined/PyGrid
+cd PyGrid
+sudo echo """
```

Comment:
So in theory, we won't be doing this because nothing will be hosted at the same private IP address.

Reply:
Yeah this is a bit outdated as we discussed :)